_IBC/terra-migaloo.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "terra",
```

Comment:
please reverse the order of chain_1 and chain_2 (because terra comes later, alphabetically). And the file name, too

Reply:
Done